is-with-analytics/simple/deployment.yaml

```diff
@@ -0,0 +1,336 @@
+            runAsUser: 802
+          ports:
+            - containerPort: 9763
+              protocol: TCP
+            - containerPort: 9443
+              protocol: TCP
+      initContainers:
+        - name: wso2-is-init
+          image: busybox
+          command: ['sh', '-c', 'until nslookup wso2is-with-analytics-is-service; do echo waiting for wso2-is-service; sleep 2; done']
+      serviceAccountName: "wso2svc-account"
+      imagePullSecrets:
+        - name: wso2creds
+
+---
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: wso2is-with-analytics-is-analytics-deployment
+spec:
+  replicas: 1
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deployment: wso2is-with-analytics-is-analytics
+        node: wso2is-with-analytics-is-analytics
+    spec:
+      containers:
+        - name: wso2is-with-analytics-is-analytics
+          image: docker.wso2.com/wso2is-analytics-worker:5.7.0
+          env:
+            -
+              name: NODE_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+          resources:
+            limits:
+              memory: "4Gi"
+            requests:
+              memory: "4Gi"
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 9090
+            initialDelaySeconds: 200
+            periodSeconds: 10
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 7712
+            initialDelaySeconds: 200
+            periodSeconds: 10
+          lifecycle:
+            preStop:
+              exec:
+                command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/worker.sh stop']
+          imagePullPolicy: Always
+          securityContext:
+            runAsUser: 802
+          ports:
+            -
+              containerPort: 9090
+              protocol: TCP
+            -
+              containerPort: 9091
+              protocol: TCP
+            -
+              containerPort: 9543
+              protocol: TCP
+            -
+              containerPort: 9711
+              protocol: TCP
+            -
+              containerPort: 9611
+              protocol: TCP
+            -
+              containerPort: 7712
+              protocol: TCP
+            -
+              containerPort: 7612
+              protocol: TCP
+            -
+              containerPort: 7070
+              protocol: TCP
+            -
+              containerPort: 7443
+              protocol: TCP
+      initContainers:
+        - name: wso2-analytics-init
+          image: busybox
+          command: ['sh', '-c', 'until nslookup wso2is-with-analytics-is-analytics-service; do echo waiting for wso2-is-analytics-service; sleep 2; done']
```

Comment:
We don't need an init container for this right since this doesn't depend on other services. Else we can take Mysql service as a dependency.